logger/buffered_logger.go

```diff
@@ -110,6 +126,7 @@ func (bl *bufferedLogger) Start(
 		pipe := p
 
 		errGroup.Go(func() error {
+			debug.SendEventsToLog(DaemonName, fmt.Sprintf("Reading logs from pipe %s", source), debug.DEBUG, 0)
```

Comment:
Any possibility that this logging introduces possible security-related log messages?

Reply:
Yes it's the name.

ie this is from one fargate task
```
...
Reading logs from pipe stdout
Reading logs from pipe stderr
...
```